### DISCUSSION TITLE
How to hide toolbar in Rich Text Editor?

### DISCUSSION THREAD

**Discussion by atumas-bananamilk** [mantinedev/mantine#2705] [Q&A] [answered]
### What package has an issue

@mantine/rte

### Describe the bug

How to hide the toolbar (without readonly)?

Currently I use this code:

`<CaptionEditor controls={[]} />`

And I get this:

![Screenshot 2022-10-13 at 19 09 03](https://user-images.githubusercontent.com/22228431/195649418-04e4b4ea-f6c7-4a89-b5d2-eeec50e895ea.png)

### What version of @mantine/hooks page do you have in package.json?

5.1.5

### If possible, please include a link to a codesandbox with the reproduced problem

-

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

No

### Possible fix

-

**Comment by rtivital** [comment, marked as answer]
https://codesandbox.io/s/flamboyant-andras-47quqc?file=/src/App.tsx
